packages/ws-worker/test/cli.test.ts

```diff
@@ -0,0 +1,92 @@
+import test from 'ava';
+import { parseArgs } from '../src/cli';
+import { LogLevel } from '@openfn/logger';
+
+
+const resetEnv = (env: NodeJS.ProcessEnv) => {
+    process.env = { ...env };
+};
+
+test.beforeEach(t => {
+    // Store original environment variables
+    t.context = { ...process.env };
+});
+
+test.afterEach(t => {
+    // Restore original environment variables
+    resetEnv(t.context as NodeJS.ProcessEnv);
+});
+
+test('parseArgs should parse command line arguments correctly', t => {
+    const argv = ['node', 'cli.js', '--port', '3000', '--log', 'info', '--max-run-duration-seconds', '120'];
```

Comment:
I would have written the command as a string and then split it before calling parse args. It would make the test easier to to read and write:
```
const argv = 'node cli.js --port 3000 --log info --max-run-duration-seconds 120'
const args = parseArgs(argv).split(" ");
```

Reply:
I believe the usage would be:
```
    const argv = 'node cli.js --port 3000 --log info --max-run-duration-seconds 120'
    const args = parseArgs(argv.split(' '))
```